kubernetes-manifests/redis.yaml

```diff
@@ -32,8 +32,8 @@ spec:
         - containerPort: 6379
         readinessProbe:
           periodSeconds: 5
-          tcpSocket:
-            port: 6379
+          exec:
+            command: ['/bin/sh', '-c', 'redis-cli -h $(hostname) ping']
```

Comment:
I think you can probably use localhost as hostname, in that case you would not need /bin/sh -c

also, why make this change? does redis somehow start up on port 6379, and not be ready sometimes?

Reply:
I had a theory that the redis was reporting ready before it was fully ready (based on the cartservice crashloops), but I'm not sure if that's true anymore.

I thought I'd keep this since it feels more accurate to ping with redis-cli directly, I'm not sure if there are any actual functional differences, so I could take or leave this change